src/app/app-routing.module.ts

```diff
@@ -10,6 +10,10 @@ const routes: Routes = [
   { 
     path: 'dashboard', 
     loadChildren: () => import('./dashboard/dashboard.module').then(m => m.DashboardModule) 
+  },
+  {
+    path: 'persons/:id/experiences',
```

Comment:
```suggestion
    path: 'persons',
```

this module will expose his routing module. See recipe here https://angular.io/guide/lazy-loading-ngmodules